config/routes.rb

```diff
@@ -22,8 +22,9 @@ def matches?(request)
 
     constraints account: /[^\/?]+/ do
       constraints authenticator: /authn-?[^\/]*/, id: /[^\/?]+/ do
+        get '/authn-jwt/:service_id/:account(/:id)/status' => 'authenticate#authn_jwt_status'
```

Comment:
What does `:id` stand for there?

Reply:
`:id` is identity same as the `:id` in the authenticate route

Reply:
Why it is optional ?